Sources/ArcGISToolkit/Components/FeatureTemplatePicker/FeatureTemplatePicker.swift

```diff
@@ -0,0 +1,364 @@
+            bundle: .toolkitModule,
+            comment: """
+                 Details for showing a view that tells the user there are no feature templates
+                 available.
+                 """
+        )
+    }
+    static var searchTemplatesPrompt: String {
+        String(
+            localized: "Search templates",
+            bundle: .toolkitModule,
+            comment: """
+                 A prompt in the search templates text box that instructs the user that they
+                 can type in the field to search for templates.
+                 """
+        )
+    }
+    static var noFeatureTemplatesFoundTitle: String {
+        String(
+            localized: "Nothing Found",
+            bundle: .toolkitModule,
+            comment: """
+                 A title for showing a view that tells the user there were no feature templates
+                 found that match their search criteria.
+                 """
+        )
+    }
+    static var noFeatureTemplatesFoundDetail: String {
+        String(
+            localized: "There were no feature templates found that match the search criteria.",
+            bundle: .toolkitModule,
+            comment: """
+                 Details for showing a view that tells the user there were no feature templates
+                 found that match their search criteria.
+                 """
+        )
+    }
+}
+
+/// View of a feature template.
+private struct FeatureTemplateView: View {
+    let info: FeatureTemplateInfo
+    @Binding var selection: FeatureTemplateInfo?
+    
+    // The maximum size of the image when the font is body.
+    @ScaledMetric(relativeTo: .body) var maxImageSize = 44
+    
+    var body: some View {
+        HStack {
+            Label {
+                Text(info.template.name)
+                    .lineLimit(1)
+            } icon: {
+                if let image = info.image {
+                    if #available(iOS 17.0, *) {
+                        Image(uiImage: image)
+                            .resizable()
+                            .scaledToFit()
+                            .containerRelativeFrame(.horizontal) { size, axis in
+                                let maxSize = maxImageSize
+                                if image.size.width > maxSize {
+                                    // Limit icon to max size.
+                                    return maxSize
+                                } else {
+                                    // But don't restrict how small the icon can be.
+                                    return image.size.width
+                                }
+                            }
+                    } else {
+                        // Fallback on earlier versions
+                    }
+                } else {
+                    Image(systemName: "minus")
+                        .foregroundStyle(.secondary)
+                }
+            }
```

Comment:
Should the conditions on 155 and 156 be combined so that iOS 16 at least gets the minus icon?

```suggestion
                if let image = info.image, #available(iOS 17.0, *) {
                        Image(uiImage: image)
                            .resizable()
                            .scaledToFit()
                            .containerRelativeFrame(.horizontal) { size, axis in
                                let maxSize = maxImageSize
                                if image.size.width > maxSize {
                                    // Limit icon to max size.
                                    return maxSize
                                } else {
                                    // But don't restrict how small the icon can be.
                                    return image.size.width
                                }
                            }
                } else {
                    Image(systemName: "minus")
                        .foregroundStyle(.secondary)
                }
            }
```

Reply:
I still need to find a solution for iOS 16, working on that now.